bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/SyncThread.java

```diff
@@ -133,7 +133,7 @@ public Future requestFlush() {
     private void flush() {
         Checkpoint checkpoint = checkpointSource.newCheckpoint();
```

Comment:
We create a checkpoint at line 134, and write the checkpoint lastMark position into ledger directory at line 152. There is a gap that the actually flushed lastMark position is less than write cache flushed position.